src/video-player.js

```diff
@@ -317,6 +370,11 @@ class VideoPlayer extends Utils.mixin(Eventable) {
     return this;
   }
 
+  playItem(item) {
```

Comment:
I would keep `playItem` inside `playlist` and not expose it outside in the VideoPlayer.